### ISSUE TITLE
The image in list podcasts misaligned if the name has two lines

### ISSUE THREAD

**Issue by yg** [thepracticaldev/dev.to#7752]
**Describe the bug**

The image in list podcasts misaligned if the name has two lines

**Screenshots**

![Screenshot](https://user-images.githubusercontent.com/61746440/81465692-69ffca00-91e9-11ea-9828-fe1dc5f1f998.png)

**Desktop (please complete the following information):**

- OS: Ubuntu 19.04
- Browser: Chrome Beta
- Version: 83.0.4103.23 (Official Build) beta (64-bit)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by catalinpit** [comment]
Hello, can I pick this one, please?

**Comment by Ridhwana** [comment]
Hi @catalinpit sure go for it :) I've assigned you the issue.

**Comment by icncsx** [comment]
This seems to be a non-issue for me. Perhaps another PR inadvertently fixed the alignment issue. I noticed we use flexbox 😃 (not sure if that was there before).

I would say this is pretty well aligned.

![Screenshot from 2020-06-05 15-12-39](https://user-images.githubusercontent.com/60417145/83914339-0e205680-a73f-11ea-9ad1-91062006a7c9.png)

@Ridhwana do you see the same thing on your end?

**Comment by Ridhwana** [comment]
This looks good now - as @icncsx pointed out, this must have been a side effect from another PR. Yay for small wins 😊  

<img width="293" alt="Screenshot 2020-06-08 at 11 10 33" src="https://user-images.githubusercontent.com/2786819/84013125-cd138680-a978-11ea-88bc-43afaf57325c.png">

@catalinpit I'll be closing the issue. Thanks!
